dsl/attribute-conversion.js

```diff
@@ -177,19 +180,17 @@ class AttributeConversionSupport {
   _convertAttribute(node, attribute) {
     let isComplex = typeof attribute !== 'string';
     let name = isComplex ? attribute.name : attribute;
-    let values = isComplex ? attribute.values : [];
```

Comment:
the var was created unnecessarily. It is only used on line 193, and an `isComplex` check already occurs right before its usage, so the only state we'd ever arrive in is `attribute.values`.